docs/rfcs/016-user-and-database-management.md

```diff
@@ -0,0 +1,68 @@
+## Zentih user and database management RFC
+
+### CREATE USER
+
+CREATE USER is only allowed from console, 
+because it requires generation of the access token and other complex logic.
```

Comment:
I'd say more exact. Because console is involved in the auth process, when users connect via proxy. It's much handier to have a single source of truth.